examples/fluids/qfunctions/newtonian.h

```diff
@@ -249,6 +250,16 @@ CEED_QFUNCTION_HELPER int IFunction_Newtonian(void *ctx, CeedInt Q, const CeedSc
     UnpackState_U(s_dot.U, U_dot);
 
     for (CeedInt j = 0; j < 5; j++) v[j][i] = wdetJ * (U_dot[j] - body_force[j]);
+    if (context->idl_enable) {
+      const CeedScalar sigma     = LinearRampCoefficient(1 / context->idl_decay_time, context->idl_length, context->idl_start, x_i[0]);
```

Comment:
We probably want to invert as preprocessing so we can handle `-1` as disable. That is, `-idl_decay_time 0` should not cause a divide by zero exception.